contact.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Me</title>
+</head>
+<body>
+  <h1>Contact Me</h1>
+  <!-- 
+
+    INSTRUCTIONS
+    1. All input fields must be validated with required attribute
+    2. The email input field type should be email
+    3. The submit button type should be submit
+
+   -->
+  <form>
+    <input type="text" placeholder="First Name" /><br>
+    <input type="text" placeholder="Last Name" /><br>
+    <input type="text" placeholder="Email" /><br>
```

Comment:
your input type of email should be "email" kindly correct it according to the requirement